plugin.xml

```diff
@@ -5,13 +5,22 @@
     <label lang="de">Backend Bestellungen</label>
     <label lang="en">Backend orders</label>
 
-    <version>3.1.0</version>
+    <version>3.1.1</version>
     <copyright>(c) by shopware AG</copyright>
     <license>proprietary</license>
     <link>https://store.shopware.com</link>
     <author>shopware AG</author>
     <compatibility minVersion="5.6.0" maxVersion="5.99.99"/>
 
+    <changelog version="3.1.1">
+        <changes lang="de">
+            PT-13118 - Verwendung des EmailValidatorInterface anstatt des EmailValidator Services, was das Auslösen eines Fehlers verhindert;
```

Comment:
please remember that those changelogs should not be to technical, as they are read by merchants. 

```suggestion
            PT-13118 - Verbessert Kompatibilität mit anderen Plugins;
```